worker/src/main/java/gov/cms/ab2d/worker/processor/ContractData.java

```diff
@@ -16,11 +18,17 @@
 
     private final Contract contract;
     private final Job job;
-    private final StreamHelper streamHelper;
-    private final List<Future<EobSearchResult>> eobRequestHandles = new LinkedList<>();
+    private final List<Future<ProgressTrackerUpdate>> eobRequestHandles = new LinkedList<>();
+    private Future<Integer> aggregatorHandle;
+    private Map<Integer, Integer> hashBeneMapper = new HashMap();
 
-    public void addEobRequestHandle(Future<EobSearchResult> eobRequestHandle) {
+    public void addEobRequestHandle(Future<ProgressTrackerUpdate> eobRequestHandle, int numBenes) {
         eobRequestHandles.add(eobRequestHandle);
+        hashBeneMapper.put(eobRequestHandle.hashCode(), numBenes);
```

Comment:
Really confused by this and why it is necessary. Could use a run through and some documentation?